src/observatoryData.js

```diff
@@ -0,0 +1,76 @@
+const observatoryData = {
+  i: {
+    id: "i",
+    logo_filename: "logo.svg",
+    thumb_filename: "thumb.jpeg",
+    video_url_id: "288344114",
+    sequence: {
+      a: "a.jpg",
+      b: "b.jpg",
+      c: "c.jpg",
+      d: "d.jpg",
+      e: "e.jpg",
+      f: "f.jpg",
+    },
+  },
+  ii: {
+    id: "ii",
+    logo_filename: "logo.svg",
+    thumb_filename: "thumb.jpeg",
+    video_url_id: "288344114",
+    sequence: {
```

Comment:
Each sequence slide needs to have 
- Copy line 1
- Copy line 2
- Image

Reply:
I have all this copy I can make a commit after you set tup the structure to update it to be correct